package.json

```diff
@@ -39,6 +39,7 @@
     "simple-password-tester": "^1.0.0",
     "underscore": "^1.8.3",
     "uuid": "^3.1.0",
+    "webdriverjs-retry": "^0.2.0",
```

Comment:
Firstly this should be in "devDependencies" not "dependencies". Secondly, I'm not sure it's used at all?

Reply:
will remove this

Reply:
Cool. Be sure to revert the package-lock.json file too...